### DISCUSSION TITLE
CryptoSmite Cant Powerwash

### DISCUSSION THREAD

**Discussion by 28btalavera** [3kh0/ext-remover#1066] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE]
The keyboard shortcut for Powerwash (Ctrl + Alt + Shift + R) is not working on the login page after performing CrytoSmite. How do I powerwash it to be able to login?

**Comment by S-PScripts** [comment]
closed unless 28btalavera wants to reopen
